uniffi_bindgen/src/scaffolding/templates/ObjectTemplate.rs

```diff
@@ -57,12 +56,11 @@ pub extern "C" fn {{ ffi_free.name() }}(ptr: *const std::os::raw::c_void) {
 {%- endfor %}
 
 {%- for meth in obj.methods() %}
-    #[allow(clippy::all)]
     #[doc(hidden)]
     #[no_mangle]
     pub extern "C" fn {{ meth.ffi_func().name() }}(
         {%- call rs::arg_list_ffi_decl(meth.ffi_func()) %}
-    ) -> {% call rs::return_type_func(meth) %} {
+    ) {% call rs::return_value(meth) %} {
```

Comment:
naming nit: this doesn't really represent a return *value*, but then again I don't have a good name for the thing it does represent. Maybe "return signature"?